src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -52,20 +64,44 @@ Error({Kind:ErrorKind.InvalidArgument})
 Blank()
 
 >> Text(ParseJSON("true"))
-Error({Kind:ErrorKind.InvalidArgument})
+"true"
 
 >> Text(ParseJSON("false"))
-Error({Kind:ErrorKind.InvalidArgument})
+"false"
 
 >> Text(ParseJSON("5"))
-Error({Kind:ErrorKind.InvalidArgument})
+"5"
 
 >> Boolean(ParseJSON("true"))
 true
 
 >> Boolean(ParseJSON("false"))
 false
 
+>> Boolean(ParseJSON("""true"""))
+true
+
+>> Boolean(ParseJSON("""false"""))
+false
+
+>> Boolean(ParseJSON("""tRuE"""))
+true
+
+>> Boolean(ParseJSON("""fAlSe"""))
+false
+
+>> Boolean(ParseJSON("""F"""))
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> Boolean(ParseJSON("1"))
+true
+
+>> Boolean(ParseJSON("0"))
+false
+
+>> Boolean(ParseJSON("5"))
+true
```

Comment:
@jorisdg  - here's your test. 
Does this mean that Boolean(5) is also true?

Reply:
#489 is something that will need to be fixed in a future PR. It is not a breaking change to enable Boolean(true) because it was a compile error before, and erroneous cases are always forward-compatible.

Reply:
Boolean(2) already has a test, line 46 of Boolean.txt

Reply:
Agreed - 489 is separate and doesn't need to be handled in this PR. Just tagging this since it's related.